src/Pages/ProductList/ProductListMain/MainHeader/HeaderOption/BedroomModal/BedroomModal.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+
+export default class BedroomModal extends Component {
+  render() {
+    return <BedroomModalContainer />;
```

Comment:
```suggestion
    return <ModalContainer />;
```
이것도 충분히 뜻이 전달 될거 같습니다

Reply:
수정 완료